Automattic/wp-calypso
# Extensions: Zoninator - Add Edit Zone tab

This PR implements the tab for managing zone settings based on the form in WP Admin.  
The UI is functional with the exception of fetching/saving actual settings.

**Calypso:**

![screen shot 2017-07-24 at 11 28 01](https://user-images.githubusercontent.com/8056203/28518868-c7621cb0-7069-11e7-9ea6-4a4140b3fb0f.png)

**WP Admin:**

![screen shot 2017-07-24 at 12 15 44](https://user-images.githubusercontent.com/8056203/28518901-e8cea4cc-7069-11e7-8515-e0c9b79b8ba5.png)

# Testing:

- Go to Zoninator settings and select any zone. Ensure the 'edit zone' tab is loaded.
- Select a post from 'Recent posts' dropdown and make sure it gets added to the zone.
- Search for a post using search and see if the autocomplete dropdown with your posts appears.
- Select a post by using the mouse or arrow and enter keys. Make sure it is added to the zone as well.
- Try dragging and rearranging posts in the zone.
- Click 'Remove' on a post and make sure it's removed.

**Note:** Manage content section should only be visible after adding at least one post to the zone.
